### PR TITLE
Kinarashah certs

### PR DESCRIPTION
**Pull request by MBishop17**
initial draft of certificate tutorial. related to #248

### CODE REVIEW

**Comment by MBishop17** [comment]
related to #375

File: content/rancher/v2.x/en/tasks/projects/add-ssl-certificates/_index.md [hunk 1]
```diff
@@ -3,4 +3,40 @@ title: Adding SSL Certificates
 weight: 
 draft: true
 ---
-Coming Soon
+When you create an ingress within Rancher/Kubernetes, you must provide it with a TLS private key and certificate, which are used to encrypt and decrypt communications that come through the ingress. You can make certificates available for ingress use by navigating to its project or namespace, and then uploading the certificate.
+
+SSL certificates to either projects, namespaces, or both. A project scoped certificate will be available in all its namespaces.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/add-ssl-certificates/_index.md]: The first sentence seems to be missing some kind of verb.

File: content/rancher/v2.x/en/tasks/projects/add-ssl-certificates/_index.md [hunk 2]
```diff
@@ -3,4 +3,40 @@ title: Adding SSL Certificates
...
+
+1. From the main menu, select **Resources > Certificates**. Click **Add Certificate**.
+
+1. Enter a **Name** for the certificate.
+
+    >**Note:** Kubernetes classifies SSL certificates as [secrets](https://kubernetes.io/docs/concepts/configuration/secret/), and no two secrets in a project or namespace can have duplicate names. Therefore, to prevent conflicts, your SSL certificate must have a unique name among the other certificates, ConfigMaps, registries, and secrets within your project/workspace.
+
+1. Select the **Scope** of the certificate.
+
+    - **Available to all namespaces in this project:** The certificate is available for deployments to all namespaces in the project.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/add-ssl-certificates/_index.md]: Update to ```
The certificate is available for any deployment in any namespaces in the project.```

File: content/rancher/v2.x/en/tasks/projects/add-ssl-certificates/_index.md [hunk 3]
```diff
@@ -3,4 +3,40 @@ title: Adding SSL Certificates
...
+
+    - **Available to a single namespace:** The certificate is only available for deployments to one namespace. If you choose this option, select a **Namespace** from the drop-down list or click **Add to a new namespace** to add the certificate to a namespace you create on the fly.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/projects/add-ssl-certificates/_index.md]: Update to:

```
The certificate is only available for the deployments in one namespace.

**Pull request merged**